app/Controllers/SettingsController.php

```
@@ -757,15 +757,15 @@ public function updateLabels(Request $request, Response $response, mysqli $db):
             if ($width >= 10 && $width <= 100 && $height >= 10 && $height <= 100) {
-                $formatName = "{$width}×{$height}mm";
-
+                // NOTE: label.format_name is intentionally NOT persisted. It is a
+                // purely derived display string ("{w}×{h}mm") that nothing reads —
+                // the label PDF (LibriController) and the labels tab both work off
+                // width/height. Persisting it was a dead write; drop it.
                 $repository->set('label', 'width', (string) $width);
                 $repository->set('label', 'height', (string) $height);
-                $repository->set('label', 'format_name', $formatName);
 
                 ConfigStore::set('label.width', $width);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\ConfigStore' in method 'updateLabels'. (undefined)

(StaticAccess)